composer.json

```
@@ -9,7 +9,8 @@
   "prefer-stable": true,
   "minimum-stability": "dev",
   "require": {
-    "pimcore/pimcore": "11.x-dev as 11.99.9"
+    "pimcore/pimcore": "11.x-dev as 11.99.9",
```

```suggestion
    "pimcore/pimcore": "^11.0.0",
```
this will anyway install the dev version with dev stability...right?

---

likely yes 🤔 but need to clean up everywhere